### PR TITLE
antidote: init at 1.8.6

### PR DESCRIPTION
**Pull request by hitsmaxft**
###### Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [x] aarch64-linux
  - [X] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

antidote https://getantidote.github.io/ 

> Antidote is a Zsh plugin manager made from the ground up thinking about performance.
> It is fast because it can do things concurrently, and generates an ultra-fast static plugin file that you can easily load from your Zsh config.
> It is written natively in Zsh, is well tested, and picks up where [Antibody](https://github.com/getantibody/antibody) left off.

### CODE REVIEW

File: maintainers/maintainer-list.nix
Comment (kilianar) [inline maintainers/maintainer-list.nix:6374]: There is a trailing whitespace causing the EditorConfig check to fail.
```suggestion
    githubId = 352727;
```

Additionally, entries in the list should not be separated by empty lines.

File: pkgs/shells/zsh/antidote/default.nix
Comment (kilianar) [inline pkgs/shells/zsh/antidote/default.nix:25]: > Don’t include a period at the end. Don’t include newline characters. Capitalise the first character. For brevity, don’t repeat the name of package — just describe what it does. 

https://nixos.org/manual/nixpkgs/unstable/#var-meta-description

File: pkgs/shells/zsh/antidote/default.nix
Comment (kilianar) [inline pkgs/shells/zsh/antidote/default.nix:1]:
```suggestion
```

File: pkgs/shells/zsh/antidote/default.nix
Comment (kilianar) [inline pkgs/shells/zsh/antidote/default.nix:12]: You could use a SRI hash
```suggestion
    hash = "sha256-CcWEXvz1TB6LFu9qvkVB1LJsa68grK16VqjUTiuVG/c=";

```

File: pkgs/shells/zsh/antidote/default.nix
Comment (kilianar) [inline pkgs/shells/zsh/antidote/default.nix:4]:
```suggestion
{ lib, stdenvNoCC, fetchFromGitHub }:

stdenvNoCC.mkDerivation rec {
```
Since this derivation doesn't involve any building and solely entails file copying, it would be more appropriate to utilize `stdenvNoCC` instead.

**Comment by kilianar** [comment]
You should consider adding yourself as a [package maintainer in the meta attributes](https://nixos.org/manual/nixpkgs/unstable/#var-meta-maintainers).

File: maintainers/maintainer-list.nix
Comment (drupol) [inline maintainers/maintainer-list.nix:6359]: This is unrelated changes and needs to be removed, or at least make it consistent with others.

File: pkgs/shells/zsh/antidote/default.nix
Comment (drupol) [inline pkgs/shells/zsh/antidote/default.nix:19]: Please remove the empty lines here.

File: pkgs/shells/zsh/antidote/default.nix
Comment (drupol) [inline pkgs/shells/zsh/antidote/default.nix:24]: There's no need to create the `outdir` variable here.

File: pkgs/top-level/all-packages.nix
Comment (drupol) [inline pkgs/top-level/all-packages.nix:14308]: This needs to be added at the right place. Even if this file is not sorted perfectly, adding `antidote` in the middle of two derivations starting with `z` is totally wrong.

**Review by drupol — CHANGES_REQUESTED** [review]

File: pkgs/shells/zsh/antidote/default.nix
Comment (hitsmaxft) [reply pkgs/shells/zsh/antidote/default.nix]: fixed [62ac840f8d76c44aef341045dc4aff02aa5ed104](https://github.com/NixOS/nixpkgs/pull/234462/commits/62ac840f8d76c44aef341045dc4aff02aa5ed104)

File: pkgs/shells/zsh/antidote/default.nix
Comment (drupol) [inline pkgs/shells/zsh/antidote/default.nix:31]:
```suggestion
    maintainers = [ lib.maintainers.hitsmaxft ];
```

File: pkgs/shells/zsh/antidote/default.nix
Comment (drupol) [inline pkgs/shells/zsh/antidote/default.nix]: Why this empty line here?

**Review by drupol — CHANGES_REQUESTED** [review]

File: pkgs/shells/zsh/antidote/default.nix
Comment (hitsmaxft) [reply pkgs/shells/zsh/antidote/default.nix]: fixed

**Review by drupol — CHANGES_REQUESTED** [review]
Thanks for fixing everything. Squash all the commits into one single commit and I'll merge this.

**Comment by drupol** [comment]
The commit message is not correctly formatted.

It must follow our guidelines and therefore be `antidote: init at 1.8.6`

**Comment by hitsmaxft** [comment]
> antidote: init at 1.8.6

https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
sorry 
I just copy from others  lol, now it fixed.

**Review by drupol — APPROVED** [review]

File: pkgs/shells/zsh/antidote/default.nix
Comment (drupol) [inline pkgs/shells/zsh/antidote/default.nix:31]:
```suggestion
    maintainers = [ lib.maintainers.hitsmaxft ];
```

**Review by drupol — CHANGES_REQUESTED** [review]
Eval is broken because of that.

**Comment by hitsmaxft** [comment]
> Eval is broken because of that.

`nix-build $(pwd) -A antidote --show-trace ` is ok, how to reproduce it locally

**Review by drupol — CHANGES_REQUESTED** [review]
Please squash your commits and let's wait for the end of all the evaluation before merging.

**Comment by hitsmaxft** [comment]
> Please squash your commits and let's wait for the end of all the evaluation before merging.

done

**Comment by drupol** [comment]
@GrahamcOfBorg build antidote

**Review by drupol — APPROVED** [review]

**Pull request merged**
